### PR TITLE
Feature/profile

### PR DESCRIPTION
**Pull request by Vansh353**

### CODE REVIEW

File: dtos/user_models.py [hunk 1]
```diff
@@ -11,4 +11,9 @@ class UserSignupDto(BaseModel):
...
     _name = validator("name", allow_reuse=True)(validate_name)
     _password = validator("password", allow_reuse=True)(validate_password)
-    _email = validator("email", allow_reuse=True)(validate_email)
\ No newline at end of file
+    _email = validator("email", allow_reuse=True)(validate_email)
+    
+class ProfileDto(BaseModel):
+    name: Optional[str]=None
+    phonenumber: Optional[int]=None
+    address: Optional[str]=None
```
Comment (niyati-doshi20) [inline dtos/user_models.py:15]: Are we using this ProfileDTO anywhere?

File: models/order_table.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+order_items_table = Table(
+    "orderitem",
+    meta,
+    Column("id", Integer, primary_key=True, autoincrement=True),
+    Column("order_id", Integer, ForeignKey('order.id'), nullable=False),
+    Column("product_id", Integer, ForeignKey('product.id'), nullable=False),
+    Column("quantity", Integer, nullable=False),
+    Column("status", Enum(OrderStatus), default=OrderStatus.Pending, nullable=False),
+    Column("old_price", DECIMAL(10, 2), nullable=False),
```
Comment (niyati-doshi20) [inline models/order_table.py]: Make this old_price to price.

File: models/user_table.py [hunk 3]
```diff
@@ -15,6 +15,8 @@
     Column("name", Text, nullable=False),
     Column("email", Text,unique=True,nullable=False),
     Column("password", Text,nullable=False),
+    Column("phonenumber", Integer,nullable=True),
```
Comment (niyati-doshi20) [inline models/user_table.py]: It should be phone_number.

File: dtos/user_models.py (hunk 1 above)
Comment (Vansh353) [reply dtos/user_models.py:15]: yes in update profile

**Pull request merged**